writing-visualising/dashboards.qmd

```diff
@@ -22,10 +22,14 @@ There is also a dashboards version of the content design checklist. This checkli
 
 ---
 
-# Things to consider
```

Comment:
I think you've now moved a lot of bits all around the page - this does read better in some ways but there was an intentional structure that has been lost a bit, interested to know your thoughts on:

- The first half of the page before was intended as software-agnostic, with R Shiny specifics coming later on - we've lost the software agnostic section now as it's mixed together
- Linked to the above, user needs has dropped down, I think I'd always want that as the top part of the page
- Should perhaps we move to two separate pages, one R Shiny specific and one with more software agnostic guidance?

Reply:
Yeah, remember we talked about this before and we stuck with keeping the distinct agnostic and R-Shiny sections, but I feel like it's just got too unwieldy that way as there's lots of processes that are wrapped up together with working in R-Shiny. I think it's especially starting to feel hard to navigate as we have some agnostic stuff, lots on R-Shiny and then hardly anything on PowerBI. And the R-Shiny is only going to get bigger, which I feel will make it more and more awkward to organise and navigate.

Reply:
I think one thing I'm wary of in the combined version is that someone gets to the agnostic bit on something specific, but then doesn't look further and misses the really useful R-Shiny specific stuff.

Reply:
Could have a go at the 2 page solution and see how it looks.

Reply:
Yeah that's fair, I think you're right on the unwieldy side - could you try splitting it out into 2 pages and see how that looks? (Hopefully mostly copy / paste rather than too much effort needed)